internal/workflow/activities/bundle.go

```diff
@@ -133,7 +133,7 @@ func (a *BundleActivity) SingleFile(ctx context.Context, transferDir, key, tempF
 		return "", fmt.Errorf("error moving file (from %s to %s): %v", tempFile, path, err)
 	}
 
-	if err := os.Chmod(path, os.FileMode(0o755)); err != nil {
+	if err := os.Chmod(path, os.FileMode(0o755)); err != nil { //#nosec G301 -- Are group execute perms required?
```

Comment:
This is still needed? Looks unrelated to your changes.

Reply:
tried to run gosec locally to see if this was really need ed. could not get it to work at first. but i finally got it working and this is not necessary anymore.